format.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Formats source files according to Google's style guide. Requires clang-tidy.
+
+# Only files with these extensions will be formatted.
+EXTENSIONS="cc|h|proto"
```

Comment:
Can we format BUILD files, too?

Reply:
Not with clang-format. But we can with "buildifier". I did so in the last commit.